CONTRIBUTING.md

```diff
@@ -3,6 +3,10 @@
 Simple Icons welcomes contributions and corrections. Before contributing, please make sure you have read the guidelines below. If you decide to contribute anything, please do the following:
 
 1. Fork this repository
+1. (Optional) Clone the fork
+   ```
+   git clone --filter=tree:0 git@github.com:simple-icons/simple-icons.git
```

Comment:
I think that the `git@` protocol could be troublesome for some users (like for example me :sweat_smile:). I'm not sure if something must be configured in GIT to make this works and if brings some benefit, but I always use HTTPS:

```suggestion
   git clone --filter=tree:0 https://github.com/simple-icons/simple-icons.git
```

Reply:
Good catch, you're probably right. I'd suggest we provide both versions, that should cover 99.9% of contributors, agreed?

FYI. To use the `git@` protocol you need to configure a key (SSH or PGP) in your account (I'd recommend doing that, you can read more [in this article](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account))

Reply:
Yes, agreed with providing both versions :+1:

Reply:
Alright updated it, added comments to be extra clear about which one is which. I also decided to add a GitHub CLI option for those using it, even if just to keep in line with the clone URLs the GitHub interface gives you :arrow_down: 

![image](https://user-images.githubusercontent.com/3742559/102908501-da591400-4477-11eb-95a7-8c35e45cc542.png)

Reply:
Also, I was very surprised by how fast HTTPS was for cloning compared to SSH :open_mouth: Though using `--filter=tree:0` still cut the time in half for HTTPS.